### PR TITLE
feat: added product listing and filter

### PR DESCRIPTION
**Pull request by aryan-13**
- Added styles for filter options
- nav bar
- footer
- main product listing

Preview: https://deploy-preview-2--lunarstore.netlify.app/

### CODE REVIEW

File: css/layout/filterbar.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  }
+  
+  /* Style the checkmark/indicator */
+  .check-list .checkmark:after {
+    left: 6px;
+    top: 3px;
+    width: 4px;
+    height: 7px;
+    border: solid white;
+    border-width: 0 1.9px 1.9px 0;
```
Comment (shivampandey0) [inline css/layout/filterbar.css:93]: why not use `2px` here instead of `1.9px`

File: css/layout/home-nav.css [hunk 2]
```diff
@@ -3,7 +3,10 @@
     width: 100wh;
```
Comment (shivampandey0) [inline css/layout/home-nav.css]: `wh`? did you mean `vh`?

**Review by shivampandey0 — COMMENTED** [review]
Looks good, added some suggestions.

File: css/layout/home-nav.css (hunk 2 above)
Comment (aryan-13) [reply css/layout/home-nav.css]: Yes fixed it!

File: css/layout/filterbar.css [hunk 3]
```diff
@@ -4,7 +4,7 @@
     background-color: var(--color-grey-light-2);
     /* background-color: blanchedalmond; */
```
Comment (stutisk) [inline css/layout/filterbar.css]: can remove this  comment

**Review by stutisk — COMMENTED** [review]
looks good to merge!

File: css/layout/filterbar.css (hunk 3 above)
Comment (aryan-13) [reply css/layout/filterbar.css]: done

**Pull request merged**